protocol/bin/flow.js

```diff
@@ -1,3 +1,5 @@
+'use strict'
```

Comment:
This is ran by node which isn't Es 6. ThTs also just for es6 modules which
this isn't ( I think). When I used let it complained about not supporting
it in non strict more

On Friday, January 22, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In protocol/bin/flow.js
> <https://github.com/keybase/client/pull/1802#discussion_r50608631>:
>
> > @@ -1,3 +1,5 @@
> > +'use strict'
>
> I thought use strict wasn't required in es6?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1802/files#r50608631>.
>